test/extensions/geoip_providers/maxmind/config_test.cc

```diff
@@ -353,6 +402,34 @@ TEST(MaxmindProviderConfigTest, DifferentProviderInstancesForDifferentProtoConfi
   EXPECT_NE(driver1.get(), driver2.get());
 }
 
+using MaxmindProviderConfigDeathTest = MaxmindProviderConfigTest;
+
+TEST_F(MaxmindProviderConfigDeathTest, CountryMappingPathDoesNotExist) {
+  const auto provider_config_yaml = R"EOF(
+    common_provider_config:
+      geo_headers_to_add:
+        city: "x-geo-city"
+    city_db_path: %s
+    country_mapping_path: "/usr/share/geoip2Gcs/wrong_path.conf"
```

Comment:
since this is optional config param, we should have a test where this field is not configured.